### PR TITLE
Release of SMTCoq and Sniper

### PR DESCRIPTION
**Pull request by ckeller**
This is a release of the SMTCoq and Sniper plugins

### CODE REVIEW

File: released/packages/coq-smtcoq/coq-smtcoq.2.0+8.13/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+depends: [
+  "ocaml" {>= "4.07.1" }
+  "coq" {>= "8.13~" & < "8.14~"}
+]
+tags: [ 
+  "category:Computer Science/Decision Procedures and Certified Algorithms/Decision procedures"
+  "category:Miscellaneous/Coq Extensions"
+  "keyword: SMT"
+  "keyword: SAT"
+  "keyword: automation"
```
Comment (clarus) [inline released/packages/coq-smtcoq/coq-smtcoq.2.0+8.13/opam:22]:
```suggestion
  "keyword: automation"
  "logpath:SMTCoq"
```
to add the path to `Require` from, as suggested in https://github.com/coq/opam-coq-archive#website-and-opam-metadata

File: released/packages/coq-smtcoq/coq-smtcoq.2.0+8.13/opam [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ "Chantal Keller"
+ "Burak Ekici"
+ "Alain Mebsout"
+]
+synopsis: "A Coq plugin that checks proof witnesses coming from external SAT and SMT solvers"
+description: """
+- a certified checker for proof witnesses coming from the SAT solver ZChaff and the SMT solvers veriT and CVC4. This checker increases the confidence in these tools by checking their answers a posteriori and allows to import new theroems proved by these solvers in Coq;
+- decision procedures through new tactics that discharge some Coq goals to ZChaff, veriT, CVC4, and their combination"""
+url {
+  src: "git+https://github.com/smtcoq/smtcoq.git#SMTCoq-2.0+8.13"
```
Comment (palmskog) [inline released/packages/coq-smtcoq/coq-smtcoq.2.0+8.13/opam:39]: @ckeller the `released` repository is meant to be for tarballs/archives
```suggestion
  src: "https://github.com/smtcoq/smtcoq/archive/refs/tags/SMTCoq-2.0+8.13.tar.gz"
  checksum: "sha512=2669f7a94eec29962af463b9ebd54eba2dd52d8237be4852d40f4004fd39f89621ceefadd64b47f2f9866b4f31f8e442ff8d238fc69395f9a5e96e5347cb738c"
```

**Comment by palmskog** [comment]
@ckeller OK, I figured out what the mli problem is about. Basically, our `coq.8.13.dev` package here is broken when compared to `coq.8.13.2`. So I think we can ignore this particular error here (and possibly patch the `coq.8.13.dev` package later).

**Comment by ckeller** [comment]
> I see the following for OCaml 4.13.1:
> 
> ```
> # CAMLOPT -c -for-pack Smtcoq_plugin smtlib2/smtlib2_solver.ml
> # File "smtlib2/smtlib2_solver.ml", line 18, characters 2-21:
> # 18 |   cmd : string array;
> #        ^^^^^^^^^^^^^^^^^^^
> # Error (warning 69 [unused-field]): record field cmd is never read.
> # (However, this field is used to build or mutate values.)
> # make[2]: *** [Makefile:668: smtlib2/smtlib2_solver.cmx] Error 2
> ```

It is solved, and I have no other failing warning with Ocaml 4.13.1 on my machine. Hopefully it will go through :-)

**Comment by palmskog** [comment]
Only the `coq.8.13.dev` problem (on OCaml 4.13) remained now, so merging.

**Pull request merged**